flink-runtime/src/test/java/org/apache/flink/runtime/operators/CoGroupTaskTest.java

```diff
@@ -34,11 +34,14 @@
 import org.apache.flink.types.Value;
 import org.apache.flink.util.Collector;
 
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.TestTemplate;
 
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.fail;
+
 public class CoGroupTaskTest extends DriverTestBase<CoGroupFunction<Record, Record, Record>> {
```

Comment:
Make test classes package-visible. Others are similar, you can check it globally